activerecord/lib/active_record/connection_adapters/sqlite3/database_statements.rb

```diff
@@ -56,11 +56,15 @@ def internal_exec_query(sql, name = nil, binds = [], prepare: false, async: fals
           end
         end
 
-        def exec_delete(sql, name = "SQL", binds = []) # :nodoc:
+        def _exec_statement(sql, name = "SQL", binds = []) # :nodoc:
```

Comment:
Do we need to introduce this new method? We could rename `exec_delete` to `exec_statement` and remove all `exec_delete` references.

Reply:
After the rebase this change will no longer be necessary.

Latest WIP: https://github.com/rails/rails/compare/main...abaldwin88:wip_prototype_rebase_virtual_stored_column_returning_on_update